ci_jobs_trigger/libs/operators_iib_trigger/iib_trigger.py

```diff
@@ -108,9 +108,7 @@ def get_new_iib(config_data, logger):
     new_trigger_data = False
     data_from_file = get_iib_data_from_file(config_data=config_data)
     new_data = copy.deepcopy(data_from_file)
-    if (ci_jobs := config_data.get("ci_jobs", {})) is None:
-        logger.error(f"{LOG_PREFIX} No ci_jobs found in config")
-        return {}
+    ci_jobs = config_data["ci_jobs"]
```

Comment:
this will raise if the config does not have `ci_jobs`

Reply:
The func `get_new_iib()` is called after the check `if (ci_jobs := config_data.get("ci_jobs", {})) is None` in `fetch_update_iib_and_trigger_jobs()` so it will not raise the error